README.md

```diff
@@ -345,6 +345,13 @@ TYK_PMP_PUMPS_MONGOAGG_TYPE=mongo-pump-aggregate
 TYK_PMP_PUMPS_MONGOAGG_META_USEMIXEDCOLLECTION=true
 ```
 
+###### Self Healing
+The default max size of a document in MongoDB is 16Mb. In Mongo Aggregate, when we try to update a document, and the size reaches the limit, an error is received.
+When Self Healing is enabled and a document size hits 16Mb, Pump will create a new document with the current timestamp and reduce the aggregation time configuration by half. So, if the current_aggregation time is equal to 50, it will be reduced to 25. Also, the data that was being written, is sent to the new document created using its timestamp. 
+Now, following this example, new data will be stored into this new document, and a new document will be created every 25 minutes.
```

Comment:
For example, if the `current_aggregation_time` is configured as 50 (minutes) but the document hits the maximum size (16MB), a new document will be started and the `current_aggregation_time` will be set to 25 (minutes).